CryptoPrices/Resources/Assets.xcassets/ic_arrow_up_green.imageset/Contents.json

```diff
@@ -0,0 +1,12 @@
+{
+  "images" : [
+    {
+      "filename" : "ic_arrow_up_green.pdf",
+      "idiom" : "universal"
+    }
+  ],
+  "info" : {
+    "author" : "xcode",
+    "version" : 1
+  }
```

Comment:
Not a big deal, but for pdf, we'd prefer enabling `preserve vector data`, so we don't have to care about scaling the image :D.

Reply:
Good knowledge to know, will update 👍